codalab/apps/api/views/worksheet_views.py

```diff
@@ -275,7 +275,7 @@ def post(self, request, uuid):
 
                 # TODO: do this generally based on the CLI specs.
                 # Remove generated fields.
-                for key in ['data_size', 'created', 'time', 'memory', 'exitcode', 'actions']:
+                for key in ['data_size', 'created', 'time', 'memory', 'exitcode', 'actions', 'started', 'last_updated']:
```

Comment:
Ah, good.  Eventually, we should fix this part - make it an explicit github issue?